base/base_footer.ts

```diff
@@ -1,10 +1,9 @@
 declare module 'electron' {
-  const electron: Electron.AllElectron;
-  export = electron;
+  export = Electron;
```

Comment:
I'm concerned this isn't 100% backward compatible, the interface exposed by `AllElectron` is not the same as `Electron`.  I'll run this locally and see if it builds with my TS projects.

/cc @bpasero being the biggest TS project I know, could you validate the typings outputted by this change as well.  Would hate to break you guys 👍

Reply:
Yes, I modified the Electron namespace. https://github.com/electron/electron-typescript-definitions/pull/78/files#diff-aedc36ee7cf303f0616c4eaa52999c85R74

Reply:
@MarshallOfSound happy to try it out if there is a electron.d.ts to play with.

Reply:
@bpasero https://gist.github.com/NaridaL/fedfa9e4272683be12d172eea0a28dee

Reply:
Does not seem to produce any new errors. Is the idea to no longer use Electron.* to declare types?

Reply:
@bpasero Exactly, you should be able to do 
```
import {BrowserWindow} from 'electron'
const myBW: BrowserWindow = new BrowserWindow()
```